packages/dso-toolkit/components/Componenten/badge/badge.njk

```diff
@@ -1 +1,5 @@
 <span {{ className('dso-badge', [modifier, 'badge-' + modifier]) }}>{% if status -%}<span class="sr-only">{{ status }}: </span>{% endif -%}{{ label }}</span>
+
+<dso-badge level="{{ modifier }}">
```

Comment:
Omdat `level` optioneel is, wordt bij het renderen van de standaard variant (zonder modifier) een lege level getoond. Afvangen met het Nunjucks filter `attributes()`, zie `button.njk` als voorbeeld